### PR TITLE
Django2

### PR DESCRIPTION
**Pull request by msouff**
-Upgraded to Django 2.1.7
-Updated tests

### CODE REVIEW

**Comment by swainn** [comment]
@sdc50 @SarvaPulla - Django 2.1 in Tethys!

File: tests/apps/tethysapp-test_app/tethysapp/test_app/handoff.py [hunk 1]
```diff
@@ -11,16 +11,16 @@ def csv(request, csv_url):
     Handoff handler for csv files.
     """
     # Get a filename in the current user's workspace
-    user_workspace = request.workspace
-    filename = os.path.join(user_workspace, 'hydrograph.csv')
+    # user_workspace = request.workspace
```
Comment (sdc50) [inline tests/apps/tethysapp-test_app/tethysapp/test_app/handoff.py]: Can we delete this commented code?

File: tethys_compute/models/dask/dask_scheduler.py [hunk 2]
```diff
@@ -23,6 +23,8 @@ class DaskScheduler(Scheduler):
     heartbeat_interval = models.IntegerField(blank=True, default=0)
     dashboard = models.CharField(max_length=255, blank=True, null=True)
 
+    # objects = models.Manager()
```
Comment (sdc50) [inline tethys_compute/models/dask/dask_scheduler.py]: can we remove this?

File: docs/docs_environment.yml [hunk 3]
```diff
@@ -12,8 +12,8 @@ channels:
 
 dependencies:
 - python=3.5
```
Comment (sdc50) [inline docs/docs_environment.yml]: Can this be updated?

**Pull request merged**